### DISCUSSION TITLE
prFooter should honour productLinks

### DISCUSSION THREAD

**Discussion by TobiX** [renovatebot/renovate#28267] [Suggest an Idea]
### Tell us more.

The default for `prFooter` contains the URL https://github.com/renovatebot/renovate, see https://github.com/renovatebot/renovate/blob/c4c8f0caad2425a33358b998fd87f8da09e85820/lib/config/options/index.ts#L2185

This should probably honour the configuration option `productLinks` if `prFooter` isn't overwritten.

**Comment by rarkins** [comment]
You can change/override prFooter itself though right? So you're asking to save a configuration step by using productLinks.homepage but keeping "Renovate Bot"?

**Reply by TobiX** [reply]
Correct. I found `productLinks` in the documentation and was surprised the footer doesn't get adjusted. That took a second round-trip through the documentation to discover `prFooter` 😄 - AFAICS `productLink.homepage` is currently only used for the onboarding PR.

**Reply by rarkins** [reply]
You're welcome to submit a PR if you can get it to work. I think the lib/util/templates filtering would need to be extended to fetch productLinks from global config (exposedConfigOptions won't work).
